kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/ChecksumLogReplayMetricsTestBase.scala

```diff
@@ -31,22 +32,12 @@ import org.apache.spark.sql.delta.sources.DeltaSQLConf
  * - PandMCheckSumLogReplayMetricsSuite - tests Protocol and Metadata loading
  * - DomainMetadataCheckSumReplayMetricsSuite - tests Domain Metadata loading
  */
-trait ChecksumLogReplayMetricsTestBase extends LogReplayBaseSuite {
+trait ChecksumLogReplayMetricsTestBase extends LogReplayBaseSuite { self: AbstractTestUtils =>
 
   /////////////////////////
   // Test Helper Methods //
   /////////////////////////
 
-  // Abstract method to be implemented by concrete test classes
-  protected def loadSnapshotFieldsCheckMetrics(
```

Comment:
this was just overriden to largely be `loadPandMCheckMetrics` so removed it and let child suites override that instead if needed